xbmc/platform/android/CPUInfoAndroid.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "CPUInfoAndroid.h"
+
+#include "platform/android/activity/AndroidFeatures.h"
+#include "utils/Temperature.h"
```

Comment:
wrong include order

Reply:
how so?

Reply:
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers

Reply:
be specific

Reply:
- Own header file: `CPUInfoAndroid.h`
- Other Kodi includes (platform independent): `utils/Temperature.h`
- Other Kodi includes (platform specific): `platform/android/activity/AndroidFeatures.h`

Reply:
why do we want that?